README.md

```diff
@@ -77,3 +77,11 @@ tests:
 ```bash
 poetry run pytest
 ```
+
+#### Optional Static Type Checking
+
+The CI/CD pipeline will run mypy to check for type errors, so you don't need to run it locally, but if you want to check types locally, you can run:
```

Comment:
More precisely, `pytest` will run `mypy`, so you don't need to run `mypy` separately. it's not a distinction between local and CI.